invenio_drafts_resources/services/draft.py

```diff
@@ -56,6 +58,13 @@ def resolver(self):
             registered_only=False
         )
 
+    @property
+    def indexer(self):
+        """Factory for creating an indexer instance."""
+        return self.config.indexer_cls(
+            record_to_index=self.config.record_to_index
+        )
```

Comment:
We can eventually push this up to invenio-records-resources, but I do prefer starting with it here :+1:

Reply:
Shelved: Yes! will be refactored also along a small fix in `invenio-indexer`